modules/xmpp/moderator.js

```diff
@@ -210,11 +210,12 @@ Moderator.prototype.createConferenceIq = function() {
                 value: this.options.conference.minBitrate
             }).up();
     }
-    if (this.options.conference.openSctp !== undefined) {
+    if (this.options.conference.openBridgeChannel !== undefined) {
```

Comment:
I'm a bit lost here. I don't know where `this.options.config.conference.xxxx` comes from. Should I expect that `options.config.conference` in moderator references the same object as `options.config` in `JitsiConference.js`?